Source/WebCore/xml/XSLTUnicodeSort.cpp

```diff
@@ -72,19 +72,18 @@ void xsltUnicodeSortFunction(xsltTransformContextPtr ctxt, xmlNodePtr *sorts, in
 #ifdef XSLT_REFACTORED
```

Comment:
Do we still need this?

Reply:
I left it in there since the original code uses it.  Do you feel strongly about it?